ehr/resources/queries/study/treatment_order.query.xml

```diff
@@ -60,7 +60,7 @@
                         <columnTitle>Frequency</columnTitle>
                         <fk>
                             <fkDbSchema>ehr_lookups</fkDbSchema>
-                            <fkTable>treatment_frequency</fkTable>
+                            <fkTable>Treatment_Frequencies_Active</fkTable>
```

Comment:
This query looks to be defined via the schema browser for ONPRC. We will need to add it as a custom query in the EHR module to avoid breaking everyone else. Looks like no .query.xml is required, just the SQL:

Select * from ehr_lookups.treatment_Frequency
Where active = 'true'

Reply:
done